tests/testthat/test-adjust_n.R

```diff
@@ -0,0 +1,47 @@
+#' Tests the adjust_n_r and adjust_n_d functions with the examples from the vignette
+#' 
+#' TODO: Perhaps make a .RData file to be read in to be consistent with other testing styles?
+
+context("adjust_n - testing vignette examples")
+
+library(psychmeta)
+
+
+# adjust_n_r --------------------------------------------------------------
+
+test_that("adjust_n_r provides a correct/numerical answer", {
+
+  # Tests to see if working
+  expect_equal(adjust_n_r(r = .3, var_e = .01), 83.81, tolerance = 1e-2)
+
+  # Checks for "numeric" class
+  expect_is(adjust_n_r(r = 1, var_e = 1), "numeric")
+
+  # Checks for var_e = 0 error
+  expect_error(adjust_n_r(r = 1, var_e = 0), "`var_e` must be positive")
+})
+
+
+# adjust_n_d --------------------------------------------------------------
```

Comment:
Indicate "headings" in code using four #### before and after the text, rather than hyphens, but not necessary here. The title in `test_that()` suffices.